package.json

```
@@ -25,5 +26,8 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0"
+  },
+  "devDependencies": {
+    "ncc": "^0.3.6"
```

Am I looking at the right `ncc`? What does it have to do with the build?

---

Yes, that is the right ncc. It is a build tool that compiles the code and the dependency (from `node_modules`), therefore we don't need to include the `node_modules` folder in github and also during release. If you try to run `npm run build`, it should generate one `index.js` that is independent of `node_modules` folder.

---

I believe you meant `@vercel/ncc` instead of https://www.npmjs.com/package/ncc
```suggestion
    "@vercel/ncc": "^0.31.1",
```